documentation/docs/getting_started/python.mdx

```diff
@@ -112,3 +112,117 @@ print(f'{node_info}')
 Now that you are up and running, you can get acquainted with the library using
 its [how-to guides](../how_tos/00_run_how_tos.mdx) and the
 repository's [code examples](https://github.com/iotaledger/iota.rs/tree/develop/bindings/nodejs/examples).
+
+## Use with IOTA Library
```

Comment:
How about moving this into an own file? It adds a lot content which is not interesting for people that only want to use the stardust version

Reply:
I thought about it too. Couldn't come up with a good solution 😅
Will think about it, and change it for the other lib than too

Reply:
Moved it into the How To section